Actions improvements

LukaJuricic opened the pull request.

raunc commented on src/actions.tex:
```diff
@@ -88,26 +74,29 @@
+	\item Must select a face-up Event
 \end{itemize}
 \begin{enumerate}
 	\item Collect \ducats on the card and move Round Status marker
```
```suggestion
\end{itemize}
\begin{enumerate}
	\item \strong{Select} a face-up Event, \strong{collect} \ducats on the card and \strong{move} Round Status marker
```
In the rulebook "Select a face-up Event" is in step 1. I'd do the same here and highlight verbs in this sentence to make it more visible that there are 3 activities in this step. This seems to save 1 line, which would be enough to put back "Apply immediate effects" in Research Action below

raunc commented on src/actions.tex:
```diff
@@ -119,7 +108,6 @@
 		\subsubsection*{Standard Event effects \normal{(p.~40)}}
 		\smallheading{Gain/place Influence}
 		\begin{itemize}
-			\item 1+ Province in Area must be Owned by NPR/Vassal
```
Might be already implied by other rules, but I'd still keep it here as a reminder. Seems like there is enough space. If it doesn't fit, then I'd consider shortening statements about Research Idea (Saying "If the Idea is not on display, replace unpicked non-Basic Idea of the same type" in a single point instead of two seems to free up a line)

raunc commented on src/actions.tex:
```diff
@@ -133,7 +121,6 @@
 		\end{itemize}
 		\smallheading{Gain Royal Marriage}
 		\begin{itemize}
-			\item Target must be eligible
```
Same as above. Might be already implied by other rules, but I'd still keep it here as a reminder. If not enough space, then shortening Research Idea statement might help

raunc commented on src/actions.tex:
```diff
@@ -276,7 +264,7 @@
 				\begin{itemize}
 					\item May place the \dnpr on vacant Territor.
-					\item \hspacezero\zsavepos{arrowDistantTradeRightStart}If first \dnpr on a Distant Continent, add Trade Cards
+					\item \hspacezero\zsavepos{arrowdistantTradeStartThree}If first \dnpr on a Distant Continent, add Trade Cards
```
```suggestion
					\item \hspacezero\zsavepos{arrowDistantTradeStartThree}If first \dnpr on a Distant Continent, add Trade Cards
```
Typo in the variable name. Not very important, feel free to ignore

raunc commented on src/actions.tex:
```diff
@@ -379,31 +362,56 @@
+\actionHeading{Research Idea \normal{(X\monarchpower) (p.~12)}}
+\begin{itemize}
+	\item Mark it with a Tag
+	\item Score \p2
```
```suggestion
	\item Mark it with a Tag
	\item Apply immediate effects
	\item Score \p2
```
Let's re-add this, if possible. It seems possible to make room for it (see the comment above about selecting an Event). If it still does not fit, then feel free to ignore this comment. I might try to fit it later in case I have time, but not very important actually

raunc commented on src/actions.tex:
```diff
@@ -379,31 +362,56 @@
+	\item Mark it with a Tag
+	\item Score \p2
+	\item If ≤2 other PRs tagged it, they score \p1
+	\item If you \action{Research} another Government Form Idea while having one (p.~21)
```
```suggestion
	\item If you \action{Research} another \strong{Government Form} Idea while having one (p.~21)
```
Nice that rules about Government Forms are here now. I think it would be even better if highlighted. Then it is easier to quickly see that this part of the text can be skipped when not Researching Government Forms

raunc commented on src/actions.tex:
```diff
@@ -379,31 +362,56 @@
+			\item Score \prestige as normal for the new one
 	\end{itemize}
-\end{eubox}\end{dynamiccontents*}
+	\item If \authority ≥ 3, Imperial Subjects may not \action{Research} Government Form Ideas (p.~44)
```
```suggestion
	\item If \authority ≥ 3, \strong{Imperial Subjects} may not \action{Research} Government Form Ideas (p.~44)
```
I'd highlight "Imperial Subjects" or, alternatively, "Government Form" to make it easier to see that this can be skipped most of the time. (Highlighting both would probably be too much)

raunc commented on src/actions.tex:
```diff
@@ -421,119 +429,90 @@
 		\item If having \idea{QftNW}, your \ship
 	\end{itemize}
-	\item If first \claim on a Distant Continent, add Trade Cards\zsavepos{arrowDistantTradeLeftStartTwo}
+	\item Place Colonial \claim on Discovered Area
```
```suggestion
	\item Place \claim on Discovered Area
```
It would not be "Colonial" in case the Discovered Area contains NPR or PR Province.
However, it seems useful to have a definition of Colonial Claim somewhere. Perhaps the added definitions list in the beginning of Other Rules pages would be a good place for it

raunc commented on src/actions.tex:
```diff
@@ -631,40 +619,45 @@
-	\item If target is a Distant NPR, then must have a \claim on 1+ of target's Areas
+	\item If target is a Distant NPR, then 
+	\begin{itemize}
+		\item must have a \claim on 1+ of target's Areas
+		\item may use \colonist instead of \influence
```
```suggestion
		\item Must have a \claim on 1+ of target's Areas
		\item Use \colonist instead of \influence (p.~32)
```

raunc reviewed (commented):
Added some comments. There are a few more things, but no more time today. Will continue in a few days (if not before, then on Friday)

raunc commented:
Also noticed that `make` fails on this branch. Building files separately (e.g. `make actions.pdf`) produces pdfs correctly, but exits with an error when building `actions.pdf` or `sequence_of_play.pdf`, so I suspect it is caused by something in `eucommon.tex` that affects both of those, but didn't have time to look into it in more detail

LukaJuricic commented:
I fixed the issue in `actions.pdf`, it was due to a missing change in `eucommon.tex` introduced in the sequence of play PR.
I fixed this by introducing again the missing variable.

The `sequence_of_play.pdf` does not build since the arrows were updated on old frames/text in 922a395.
This would require either to revert the arrows, or merge the updated file from the sequence of play branch.
Best course of action would depend on how do you plan to merge everything together.

raunc commented:
> The `sequence_of_play.pdf` does not build since the arrows were updated on old frames/text in [922a395](https://github.com/raunc/eutpop-ref-sheet/commit/922a39551b3ac3277dbde817e814c49cdbcdf09f).
> This would require either to revert the arrows, or merge the updated file from the sequence of play branch.
> Best course of action would depend on how do you plan to merge everything together.

I guess rebasing to main branch (which already contains sequence of play update) would be cleanest. Already did it with the target branch of this PR. You can either rebase your branch too, or merge the target branch into yours

raunc commented on src/actions.tex:
```diff
@@ -615,10 +596,19 @@
 \actionHeadingNoSpace{Incr. Stab. \normal{(5\adminpower ± current \stability) (p.~14)}}
 \begin{itemize}
 	\item Increase \stability by 1
+	\item If Papal Controller, pay -1\adminpower
```
```suggestion
	\item If Papal Controller, pay -1\adminpower (p.~46)
```

raunc commented on src/actions.tex:
```diff
@@ -764,38 +751,72 @@
+				\item If >5 cards in discard pile, keep the top 5, shuffle others
+				\item If ≤5 cards in discard pile, shuffle all
+			\end{itemize}
+			\item Papal Controller pays -1\adminpower for actions increasing \stability
```
```suggestion
			\item Papal Controller pays -1\adminpower for actions increasing \stability (p.~46)
```

raunc replied:
The rule about Orthodox ability would also be good to add here. Perhaps there will be enough space when combining the points about countered Covert Action costs into one: "When contered, cost must be payed and Display Card uses spent"

raunc commented on src/actions.tex:
```diff
@@ -764,38 +751,72 @@
+		\smallheading{Covert Actions (\covertaction)}
 		\begin{itemize}
-			\item Must discard one in play
+			\item Can be countered by \action{Counterespion.}
```
```suggestion
			\item Can be countered by \action{Counterespionage}
```
Looks like there is enough space on this line now, so it doesn't have to be abbreviated anymore

raunc commented on src/actions.tex:
```diff
@@ -764,38 +751,72 @@
+			\end{itemize}
+			\item Fully charge the card with \cubes when played
+			\item 1+ charges are exhausted per use
+		\end{itemize}
```
```suggestion
		\begin{itemize}
			\item Max 2 in play per player (must be diff.)
			\begin{itemize}
				\item Must discard 1 of them to play another
			\end{itemize}
			\item Fully charge the card with \cubes when played
			\item 1+ charges are exhausted per use (in addition to \monarchpower cost)
		\end{itemize}
```
I'd like to clarify that action cost must still be paid, even though cubes are removed from cards. To make room for it, the point about discarding cards can be shortened

raunc commented on src/actions.tex:
```diff
@@ -631,40 +621,45 @@
-	\item No \influence on Distant Continents
-\end{itemize}
-
 \actionHeading{Forge Alliance \normal{(1-3\diplopower) (p.~15)}}
```
```suggestion
\actionHeadingNoSpace{Forge Alliance \normal{(1-3\diplopower) (p.~15)}}
```
Looks nicer if less space between the higher level heading

raunc commented on src/actions.tex:
```diff
@@ -631,40 +621,45 @@
+	\item Add 1\influence to target's Cap. Area
+\end{itemize}
+
+\actionHeadingNoSpace{Influence \normal{(1\diplopower/3\ducats per \influence) (p.~15)}}
```
```suggestion
\actionHeading{Influence \normal{(1\diplopower/3\ducats per \influence) (p.~15)}}
```
Does not need "no space" style as it does not come right after higher level heading anymore

raunc commented on src/actions.tex:
```diff
@@ -680,78 +675,70 @@
 	\item May move 1 Light \ship
 	\begin{itemize}
-		\item Only if
+		\item Only if, selecting a Trade Node
```
```suggestion
		\item Only if selecting a Trade Node where
```
This way it reads like a sentence when combined with sub-points

raunc commented on src/actions.tex:
```diff
@@ -631,40 +621,45 @@
+
+\actionHeadingNoSpace{Influence \normal{(1\diplopower/3\ducats per \influence) (p.~15)}}
+\begin{itemize}
+	\item Pay 1\diplopower or 3\ducats per \influence (min 1\diplopower)
+	\item May place \influence in Areas Adj. to your \towns, \vassals, \alliances, \marriages, \influence that were there at the start of the Turn
+	\item Max 2\influence per Area per Turn
+	\item No \influence on Distant Continents
```
Seems like there is enough room to put back "Area may contain max 5\influence" clarification

raunc commented on src/actions.tex:
```diff
@@ -680,78 +676,70 @@
 		\item Activate it (lay it on its side)
 	\end{itemize}
-	\item Calculate Tr. Power for all eligible PRs with Merch. in selec. Node and collect \ducats
+	\item Eligible PRs collect the Trade Income\zsavepos{arrowTradeStart}
```
```suggestion
	\item Eligible PRs collect Trade Income\zsavepos{arrowTradeStart}
```

raunc commented on src/actions.tex:
```diff
@@ -680,78 +676,70 @@
 
-		\subsubsection*{Trade Power (\tradepower) \normal{(p.~35)}}
+\begin{dynamiccontents*}{frameTradeIncome}\begin{eubox}{\fhTrade}
+	\subsubsection*{\zsavepos{arrowTradeEnd}Trade Income \normal{(p.~34-35)}}
```
Made minor wording adjustments and managed to squeeze some more details into this box. Here's the whole contents of the box after changes:

```
	\subsubsection*{\zsavepos{arrowTradeEnd}Trade Income \normal{(p.~34-35)}}
	\begin{itemize}
		\item Human PRs collect Trade Inc. if the Node
		\begin{itemize}
			\item Has their Merchant, \conj{and}
			\item Is \strong{eligible}: connected to Capital Area by \towns, \vassals, \influence or Light \ships (not required in Inland Trade Node's Area; Occupied \towns/\vassals count)
		\end{itemize}
		\item Expanded - red column, otherwise - green
		\item \strong{1 PR per row} may collect Trade Income. Priority (losers are pushed to lower row):
		\begin{enumerate}
			\item Highest Trade Power
			\item Active PR breaks ties as they please
		\end{enumerate}
		\item Each Pirate Ship lowers all PRs by 1 row
	\end{itemize}
	\smallheading{Secondary Trade Nodes}
	\begin{itemize}
		\item If Active PR has highest (may be tied) Trade Power in eligible Sec. Trade Node
		\begin{itemize}
			\item They may collect Income from the lowest row of the Secondary Node
			\item Other players collect income from Primary Node as normal
		\end{itemize}
		\item May not select a Sec. Node with a Pirate
	\end{itemize}
	\smallheading{Trade Power (\tradepower)}
	\begin{itemize}
		\item 1 \tradepower for your Merchant
		\item 1 \tradepower per Key Province you Own
		\item 1 \tradepower per \ship in Adj. Trade Protection slots (only for Maritime Trade Nodes)
		\item Key Prov. with "+" provide +1\tradepower per "+"
		\item Key Prov. do not count for Sec. Node
	\end{itemize}
	{\botrules
	\subsubsection*{Bot Trade \normal{(p.~4)}}
	\begin{itemize}
		\item Bots have 1 Merchant (never exhausted)
		\item When human PR \action{Trades}, Bots collect Trade Income if Node is eligible for them
		\begin{itemize}
			\item Maritime Trade Node (-1\botpower per Pirate)
			\begin{itemize}
				\item Main Map Nodes: 2\botpower
				\item Distant Nodes: 3\botpower
			\end{itemize}
			\item Inland Trade Node: 2\botpower
			\begin{itemize}
				\item Bot must have 1+ Adjacent \town
			\end{itemize}
			\item -1\botpower if hum. PR has 3+ \tradepower from Key Prov. and \idea{Mercantilism} Idea
		\end{itemize}
		\item When Bot collects Trade Income, move its Merchant to that Node	
	\end{itemize}
	}
```

LukaJuricic replied:
Very nice!

raunc commented on src/actions.tex:
```diff
@@ -680,78 +676,70 @@
 
 \actionHeading{Trade \normal{(1\diplopower) (p.~15)}}
```
I would move Trade before Influence to shorten the arrow

raunc commented on src/actions.tex:
```diff
@@ -764,38 +752,68 @@
-	\end{itemize}
+		\smallheading{Card specific rules}
+		\begin{itemize}
+			\item \action{One Step Ahead}: Cannot be played on an Army with \artillery\xspace > \xspace\cavalry (p.~24)
```
```suggestion
			\item \action{One Step Ahead}: Cannot be played on an Army with \artillery\xspace > \cavalry (p.~24)
```
Two spaces were rendered with both regular space and `\xspace` before cavalry icon

raunc commented on src/actions.tex:
```diff
@@ -615,10 +596,19 @@
 \actionHeadingNoSpace{Incr. Stab. \normal{(5\adminpower ± current \stability) (p.~14)}}
 \begin{itemize}
 	\item Increase \stability by 1
+	\item If Papal Controller, pay -1\adminpower (p.~46)
+	\item If Orthodox and current \stability is -1/0, pay -1\adminpower (p.~38)
```
I was also thinking that it might be better to write `1\adminpower less` instead of -1 in places like this (when there is enough room)

LukaJuricic replied:
I added the change where possible in the actions page.
While checking the whole document for this, I spotted some phrasing improvements in the sequence of play. There I left the "receive +/-X" form as it seemed clearer, but I added a couple of details. Let me know if it is ok to squeeze in past reviewed files changes like this if it happens.

raunc replied:
> I added the change where possible in the actions page. While checking the whole document for this, I spotted some phrasing improvements in the sequence of play. There I left the "receive +/-X" form as it seemed clearer, but I added a couple of details. Let me know if it is ok to squeeze in past reviewed files changes like this if it happens.

Yes, no problem. In case of bigger changes like moving boxes around I'd prefer a separate PR, though
